Prox/Prox/Utilities/OpenInHelper.swift

```diff
@@ -33,6 +34,15 @@ struct OpenInHelper {
         return true
     }
 
+    private static func extractHostname(fromHost host: String) -> String? {
+        let components = host.components(separatedBy: ".")
+        if components.count > 0 {
```

Comment:
Nit: `components.count` will always have at least 1 item, so you could just change this to `return components(separatedBy: ".")[1]`